src/MHTimer.js

```diff
@@ -1422,9 +1435,11 @@ function addRemind(timerRequest, message) {
         responses.unshift('Hi there! Reminders are only sent via PM, and I\'m just making sure I can PM you.');
 
     // Send notice of the update via PM.
-    message.author.send(responses.join(' ')).catch(() =>
-        Logger.error(`Reminders: notification failure for ${message.author.username}.`),
-    );
+    message.author.send(responses.join(' ')).catch(() => {
```

Comment:
This won't work the way you want it to - `send` is an async method, so `return CommandData(message, dm, success);` will execute *before* any catch clause can execute. You'll need to either `await` the send command, or return the promise and add a `then`. Either way, the call site will need to be updated to handle the async nature of the method.